static/app/components/stream/group.tsx

```diff
@@ -497,40 +496,23 @@ function BaseGroupRow({
 
       {withChart && !displayReprocessingLayout && issueTypeConfig.stats.enabled && (
         <ChartWrapper narrowGroups={narrowGroups}>
-          {organization.features.includes('issue-stream-new-events-graph') ? (
-            <GroupStatusChart
-              hideZeros
-              loading={!defined(groupStats)}
-              stats={groupStats}
-              secondaryStats={groupSecondaryStats}
-              showSecondaryPoints={showSecondaryPoints}
-              groupStatus={getBadgeProperties(group.status, group.substatus)?.status}
-              showMarkLine
-            />
-          ) : (
-            <GroupChart
-              stats={groupStats}
-              secondaryStats={groupSecondaryStats}
-              showSecondaryPoints={showSecondaryPoints}
-              showMarkLine
-            />
-          )}
+          <GroupStatusChart
+            hideZeros
+            loading={!defined(groupStats)}
+            stats={groupStats}
+            secondaryStats={groupSecondaryStats}
+            showSecondaryPoints={showSecondaryPoints}
+            groupStatus={getBadgeProperties(group.status, group.substatus)?.status}
+            showMarkLine
+          />
         </ChartWrapper>
       )}
       {displayReprocessingLayout ? (
         renderReprocessingColumns()
       ) : (
         <Fragment>
           {withColumns.includes('event') && issueTypeConfig.stats.enabled && (
-            <EventCountsWrapper
-              leftMargin={
-                organization.features.includes('issue-stream-new-events-graph')
-                  ? space(0)
-                  : space(2)
-              }
-            >
-              {groupCount}
-            </EventCountsWrapper>
+            <EventCountsWrapper leftMargin={space(0)}>{groupCount}</EventCountsWrapper>
```

Comment:
move leftMargin out of props and into the css since its now static. or remove it entirely since its zero